src/ReactiveUI/Platforms/apple-common/ObservableForPropertyBase.cs

```diff
@@ -83,6 +88,7 @@ protected void Register(Type type, string property, int affinity, Func<NSObject,
             typeProperties[property] = info;
         }
 
+#if UIKIT
```

Comment:
If this really is UIKit only, I'd be tempted to push it down into the UIKitObservableForProperty, though that may be a breaking change so not too worried.

Reply:
Yeah I figured it would be more logical to have all these together rather than having to look through multiple files with equal functionality. But I get what you're saying.